Sources/DangerSwiftPeripherySampleApp/Dummy2.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  File.swift
+//  
+//
+//  Created by 多鹿豊 on 2022/04/05.
+//
+
+import Foundation
+
+class UnusedClass1 {
```

Comment:
<!--
  0 failure: 
  1 warning:  Class 'UnusedClas...
  
  
  DangerID: danger-id-Danger;
  File: Sources/DangerSwiftPeripherySampleApp/Dummy2.swift;
  Line: 10;
-->

- :warning: Class 'UnusedClass1' is unused